src/logger.ts

```diff
@@ -0,0 +1,27 @@
+import { LogFunctionFactory, LogLevel, LogMeta } from "@graphile/logger";
+
+export { LogLevel as MigrateLogLevel } from "@graphile/logger";
+export type MigrateLogFactory = LogFunctionFactory<Record<string, unknown>>;
+export interface MigrateLogMeta extends LogMeta {
+  error?: Error;
+}
```

Comment:
I _think_ it's fine to just replace `LogMeta`:

```suggestion
export interface MigrateLogMeta {
  error?: Error;
}
```

I might be wrong.